internal/utils/flags/flags.go

```diff
@@ -18,12 +18,14 @@ type Flag interface {
 	Register(fs *pflag.FlagSet)
 }
 
-// Meta represents the metadata of a flag
+// Meta represents the metadata of a flag. If Hidden is set to true and Deprecate is not a zero value string,
```

Comment:
the docstring change isn't necessary here.  in the implementation of `pflag`, when a flag is deprecated it is _also_ hidden, so in theory both are still true

if you wanted to be nitty about it, the docstring might want to call out "when Deprecate is set, the flag will also be hidden".  but again, i think for simplicity's sake, let's just revert this change